source/systems/request.md

```diff
@@ -0,0 +1,43 @@
+
+## Manual Requests
+<!-- Put a picture of the red gear for manual requests here!-->
+What if your builder asks for oak stairs but you don’t have a carpenter yet? In that case, the request can’t be filled automatically, and the citizen will wait for you, the mayor, to fulfill the request manually. These citizens will have a red gear over their head and, in talking to them, will tell you what they need. Once you have that item they need, you can give it to them directly or put it in the Warehouse for your Couriers to deliver.
+
+## Clipboard
+
+Of course, being the busy mayor you are, you’ll want to be able to see all of your citizens’ requests without running to each one. For this, there’s the [Clipboard](../../source/items/clipboard). Simply right-click the air with your clipboard and you can see all of your citizens’ requests, no matter where you are!
+
+Some of your citizens will always have constant requests (e.g., the [Smelter](../../source/workers/smelter) requesting ores). Clicking the "Show Important Requests Only" hides these constant requests, so you can focus on the requests that bottleneck your citizens' work.
+
+## Stash/Postbox
+
+As your colony grows, so too will the number of items and blocks in your Warehouse and that your citizens can produce. Searching the Warehouse each time you want something can be tedious. Thankfully, your colony has a [Postbox](../../source/items/postbox), from which you can make requests just like your citizens! Simply place it down somewhere you can easily access and, when you want to request an item from your colony but don’t want to hunt it down, make a request at the postbox. These can range from held items to placeable blocks to any items your workers can craft.
+
+For convenience, your colony also has a [Stash](../../source/items/stash). The stash behaves like a reverse postbox: Couriers will take any items you place in the stash and put them back in the Warehouse.
+
+## Priority
+
+Finally, you can adjust the ordering of the request system by changing the *priority* in each building. Higher numbers mean higher priority, so buildings with a priority of 10/10 are taken care of before buildings with a priority of 1/10. This can help ensure the colonists doing urgent jobs are not delayed by those with less urgent jobs. The postbox and stash both have a priority of 11/10.
```

Comment:
Actually, priority is only for pickups, not deliveries (deliveries are automatically high priority). that reminds me - do you have a section/info on pickups?

Reply:
I do not, though I talked about it in the "getting started" section. What information would go there?

Reply:
Just changing the priority to explain it only counts for pickups, plus the getting started bit, should be fine imo

Reply:
Edited the "priority" section. Thoughts?